compiler/luci-interpreter/src/kernels/TransposeConv.test.cpp

```diff
@@ -26,15 +26,15 @@ namespace
 
 using namespace testing;
 
-template <typename T>
+template <typename T, typename B>
```

Comment:
It seems that B is always `float`. Is in necessary to add this template parameter?

Reply:
in Quantized case, it will the bias params are `int32`. so, for those cases, I split into T and B

Reply:
I see, thanks.